actionpack/lib/action_dispatch/middleware/cookies.rb

```diff
@@ -576,9 +593,11 @@ def initialize(parent_jar)
             "Read the upgrade documentation to learn more about this new config option."
         end
 
-        secret = key_generator.generate_key(request.encrypted_cookie_salt || "")[0, ActiveSupport::MessageEncryptor.key_len]
-        sign_secret = key_generator.generate_key(request.encrypted_signed_cookie_salt || "")
-        @encryptor = ActiveSupport::MessageEncryptor.new(secret, sign_secret, digest: digest, serializer: ActiveSupport::MessageEncryptor::NullSerializer)
+        cipher = "aes-256-gcm"
```

Comment:
We should just make this the default for `ActiveSupport::MessageEncryptor`, so we wouldn't need this line.

I'll make that part of @Assain's GSoC tasks 😉